src/node/element/path/data.rs

```diff
@@ -250,10 +249,10 @@ impl<'l> Parser<'l> {
 
 #[cfg(test)]
 mod tests {
-    use node::Value;
-    use super::{Data, Parser};
     use super::super::Command::*;
     use super::super::Position::*;
+    use super::{Data, Parser};
+    use crate::node::Value;
```

Comment:
Is it by rustfmt?

Reply:
No, It is by `cargo fix --edition` on commit d6159f4ef2c8fce81103cbc7ecabc4a0b01270b5